pkg/config/release.go

```diff
@@ -41,6 +41,25 @@ const (
 	RegistryPath = "ci-operator/step-registry"
 )
 
+type ConfigMapSource struct {
```

Comment:
Why do we need this here?

Reply:
In the current state this is more tied to the config in the openshift/release repo than to the reherarsals: it holds a path in the repository and a SHA, and it is also used by other functions in this package (i.e. the ones that detect "which jobs use a template"), so keeping it here makes sense.

In the followup I want to get rid of this entirely.